app/src/pages/admin/allUsersPage/allUsersGrid/nameColumn/nameColumn.scss

```diff
@@ -20,6 +20,16 @@
     margin-top: 15px;
   }
 }
+.name {
+  white-space: nowrap;
+  text-overflow: ellipsis;

```

Comment:
On v4 if user name is too long, it breaks into multiple rows. Do we need to change this behavior in v5?

Reply:
Our design team recommends to cut long names instead of breaking into rows